gtest/src/js/decode.js

```diff
@@ -0,0 +1,31 @@
+const fs = require("fs");
+const { CreateType, getWasmMetadata } = require("@gear-js/api");
+const yargs = require("yargs");
+
+const argv = yargs
+    .option('path', {
+        alias: 'p',
+        default: "../../../examples/target/wasm32-unknown-unknown/release/demo_meta.meta.wasm",
```

Comment:
that would be strange default, better return error if meta.wasm is not set

Reply:
Removed default option. Calling `node decode.js` without `-p` option occurs internal js error: 
`The "path" argument must be of type string or an instance of Buffer or URL. Received undefined`

Reply:
Same with other options in `decode.js` and `encode.js`